src/main/resources/assets/bloodmagic/lang/ja_JP.lang

```diff
@@ -163,6 +169,9 @@ item.BloodMagic.itemFilter.modItems.name=MODアイテムフィルタ
 item.BloodMagic.itemFilter.oreDict.name=鉱石辞書フィルタ
 
 item.BloodMagic.experienceTome.name=ペリティアの書
+item.BloodMagic.sanguineBook.name=血紅の書
+
+item.BloodMagic.livingPointUpgrade.draftAngelus.name=祈りの一飲み
```

Comment:
"Draft" ... ummm...?
Tentatively, mean "Mouthful of Angelus".
It seems to beverage.
drink this -> point 200 -> joy XD